src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -169,6 +177,35 @@ func useAllCores() {
 	runtime.GOMAXPROCS(numcores)
 }
 
+func configureLogging() *rotator.SizeRotator {
+	logdir := appdir.Logs("Lantern")
+	log.Debugf("Placing logs in %v", logdir)
+	if _, err := os.Stat(logdir); err != nil {
+		if os.IsNotExist(err) {
```

Comment:
No need.  If there's a problem writing, there'll be an error on write.  This was getting swallowed, but I fixed golog to log to stderr when it happens.

Here's an example - `Unable to log: open /usr/bin/lantern: permission denied`

Regarding the legacy Lantern version, we said on our Friday call that we would not worry about the use case of someone who had the old Lantern installing this new Lantern.  If we change our mind, this and a whole bunch of other things will need consideration.